bin/smack.py

```diff
@@ -141,6 +144,9 @@ def arguments():
   verifier_group.add_argument('--loop-limit', metavar='N', default='1', type=int,
     help='upper bound on minimum loop iterations [default: %(default)s]')
 
+  verifier_group.add_argument('--context-switches', metavar='k', default='1', type=int,
+    help='switch thread contexts in Corral (only) up to k number of times')
```

Comment:
We want this flag to make sense independently of the back-end verifier, so better not to mention Corral. (Since this number is a *bound*, and all tools support at least one context, this is not complete nonsense.) Also, this help message should be semi grammatical and natural. How about “bound the number of thread contexts” ?